### PR TITLE
Update Tooling.md

### PR DESCRIPTION
**Pull request by s12v**
The Play-Swagger plugin was renamed api-first-hand

### CODE REVIEW

File: tooling/Tooling.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The following frameworks were specifically designed to support the API First wor
 
 * [Connexion: OpenAPI First framework for Python on top of Flask](https://github.com/zalando/connexion)
 * [Friboo: utility library to write microservices in Clojure with support for Swagger and OAuth](https://github.com/zalando-stups/friboo)
-* [Play Swagger: build RESTful Play services from OpenAPI specification](https://github.com/zalando/play-swagger)
+* [Api-First-Hand: API-First Bootstrapping Tool for Swagger/OpenAPI specs](https://github.com/zalando/api-first-hand)
```
Comment (ePaul) [inline tooling/Tooling.md]: The text should maybe still note that this is for Play?

Comment (s12v) [inline tooling/Tooling.md]: @ePaul, good point

Comment (s12v) [inline tooling/Tooling.md]: @ePaul, updated

**Comment by ePaul** [comment]
:+1:

**Comment by fmueller** [comment]
👍

**Pull request merged**